### ISSUE TITLE
Dados

### ISSUE THREAD

**Issue by thiamsantos** [fastcep/fastcep#5]
Usar dados do [cepaberto](http://cepaberto.com/), estudar outras maneiras de ter um banco completo.

**Comment by rafa-acioly** [comment]
Criar um crawler no `viacep` para popular um banco é uma opção?

**Comment by thiamsantos** [comment]
> Criar um crawler no viacep para popular um banco é uma opção?

É uma opção também. A gente teria que tentar todas as possibilidades de CEP lá? Depois teria que normalizar os dados né.

**Comment by thiamsantos** [comment]
Eu achei na internet 5 bancos de cep. Eu fiz uma compilação desses 5 mais o banco do cepaberto e consegui juntar para 993775 registros. Os bancos e o compilado final pode ser visto aqui https://github.com/fastcep/cep-databases.

**Comment by rafa-acioly** [comment]
Muito bom @thiamsantos, estou criando os arquivos `.sql` para popular o banco, notei que o arquivo `.csv` que você colocou no repositório possui também números de residencias, quando eu for popular o banco vou exclui-los tudo bem?

**Comment by thiamsantos** [comment]
@rafa-acioly são todos os registros que contém o número, eu nem tinha visto isso? Você acha que seria interessante retirá-los do csv.

**Comment by rafa-acioly** [comment]
@thiamsantos sim, até porque o cep diz respeito ao endereço (rua) e não á uma residência especifica.

**Comment by thiamsantos** [comment]
@rafa-acioly realmente, nessa semana vou remove os números dos registros.

**Comment by thiamsantos** [comment]
@rafa-acioly estava pesquisando e [parece que o correto é incluir o número no logradouro](http://www.buscacep.correios.com.br/sistemas/buscacep/cepFormas.cfm). Inclusive a própria API dos correios [inclui o número da residência](http://www.buscacep.correios.com.br/sistemas/buscacep/detalhaCEP.cfm). Não sei se é bem isso mas acho que devemos deixar o número.
